docs/overview.md

```diff
@@ -73,6 +73,13 @@ Grafana Cloud Agent supports collecting logs and sending them to Loki using its
 is the official first-party log collection client created by the Loki
 developer team.
 
+## Traces
+
+Grafana Cloud Agent supports collecting traces and sending them to Tempo using its
+`tempo` subsystem.  This is done by utilizing the upstream [OpenTelmetry Collector](https://github.com/open-telemetry/opentelemetry-collector).
+The agent is capable of ingesting OpenTelemetry, OpenCensus, Jaeger or Zipkin spans.  See documentation
```

Comment:
Nice, thanks. Can we also note which non-Tempo systems the Agent can write spans to?